pages/blog/posts/json-schema-ambassadors.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'JSON Schema Ambassadors Are Here!'
+date: '2024-06-20'
+tags:
+  - News
+type: Community
+cover: /img/posts/2024/json-schema-ambassadors/cover.webp
+authors:
+  - name: Ben Hutton
+    photo: /img/avatars/benhutton.webp
+    link: https://www.twitter.com/relequestual
+    byline: JSON Schema Specification Lead @Postman
+excerpt: 'Just what is a JSON Schema Ambassador? And how can I help? Keep reading!'
+---
+
+## Ambassadors program?
+
+You may have noticed our new Ambassadors Program! If not, don't worry—we're here to explain what it is and how you can get involved.
+
+I'm personally very excited to see this program come to life. Recognizing our community champions in a meaningful and sustainable way has always been a goal. The Ambassadors Program is our first step in publicly supporting our most experienced and helpful members.
+
+We have many community members who frequently help others, provide feedback, answer questions, and debug schemas and code. They also assist newcomers in avoiding basic misconceptions. JSON Schema wouldn't be where it is today without this support.
+
+## The time is right - we are growing
+
+For the past few years, a small team has worked full time on JSON Schema, focusing on ongoing projects. Now, we have the capacity to launch new initiatives, including this Ambassadors Program. We want to better recognize and support those who are giving back to our community.
+
+We are better placed today than we ever have been to support and empower our existing ambassadors, and build a program which recognizes and supports them to further their efforts. It's always encouraging to see others as excited about JSON Schema as you are.
+
+"The new website, and assorted outreach programmes have started to connect the huge JSON Schema user base, and we've really started to see the benefit in terms of the number of people actively participating." - Matthew Adams - Founder at [Endjin](https://endjin.com) - Candidate Ambassador.
```

Comment:
Matthew is now TSC.  Wouldn't that mean he's no longer an ambassador candidate?

Reply:
For both Matthew and Juan, no, not at all. You can be both.
They have different expectations and responsibilities.
Someone who is a TSC member doesn't necessarily write lots of content or do talks.
Someone who is an Ambassador doesn't necessarily want to provide opinion on specification developments.

Reply:
They still qualify to be ambassadors. The only thing is they can't vote themselves (self-approve their PR)